decidim-core/app/models/decidim/moderation.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+module Decidim
+  # A moderation belongs to a reportable and includes many reports
+  class Moderation < ApplicationRecord
+    belongs_to :reportable, foreign_key: "decidim_reportable_id", foreign_type: "decidim_reportable_type", polymorphic: true
+    belongs_to :participatory_process, foreign_key: "decidim_participatory_process_id", class_name: Decidim::ParticipatoryProcess
+    has_many :reports, foreign_key: "decidim_moderation_id", class_name: "Decidim::Report"
+
+    validates :reportable, presence: true
```

Comment:
Because we need to find all moderations for a process regardless which models are `Reportable` or not. I'm not sure if there is a better way to do this, thoughts?